---
id: disallowed-tabindex-value
description: A rule that prohibits tabindex attribute values that are not erroneous as a specification but may cause confusion for users who use sequential focus.
---

# `disallowed-tabindex-value`

A rule that prohibits tabindex attribute values that are not erroneous as a specification but may cause confusion for users who use sequential focus.

<!-- prettier-ignore-end -->

❌ Examples of **incorrect** code for this rule

```html
<todo>Write incorrect codes</todo>
```

✅ Examples of **correct** code for this rule

```html
<todo>Write correct codes</todo>
```
